README.adoc

```diff
@@ -18,171 +18,14 @@
-ContextService, are difficult to use and require a lot of boilerplate
-code.  This spec makes it possible for thread context propagation to
-easily be done in a type-safe way, keeping boilerplate code to a
-minimum, as well as allowing for thread context propagation to be
-done automatically when using a CompletableFuture.
-
-It is also important that CompletableFutures and their dependent
-stages, and dependent stages created by those stages, and so on,
-all run on threads that are known by the container and properly
-managed by it.  This spec makes it possible to do so by providing
-an API by which the user obtains managed CompletableFuture instances
-from a managed executor.
-
-The above are proposed to be addressed within EE Concurrency under
-https://github.com/eclipse-ee4j/concurrency-api/issues/40[EE Concurrency Issue 40],
-however, it will be a while before Jakarta EE specs are able to
-make progress, leading us to address this first in MicroProfile.
-
-Goals
-
-* Proper management of CompletableFuture threads by the container.
-* Mechanism for thread context propagation to CompletableFuture
-actions that reduces the need for boilerplate code.
-* Full compatibility with EE Concurrency spec, such that proposed
-interfaces can eventually be seamlessly merged into EE Concurrency.
-
-[[proposed-solution]]
-Proposed solution
-~~~~~~~~~~~~~~~~~
-
-This spec introduces two interfaces that contain methods that we
-hope will eventually be added to Jakarta EE Concurrency.
-
-The interface, org.eclipse.microprofile.concurrent.ManagedExecutor,
-provides methods for obtaining managed instances of CompletableFuture
-which are backed by the managed executor as the default asynchronous
-execution facility and the default mechanism of defining thread
-context propagation. Similar to EE Concurrency's
-ManagedExecutorService, the MicroProfile ManagedExecutor also
-implements the Java SE java.util.concurrent.ExecutorService interface,
-using managed threads when asynchronous invocation is required
-and disallowing the same life cycle methods as ManagedExecutorService.
-It is intended that ManagedExecutor methods will one day be added
-to ManagedExecutorService, and for a single implementation to be
-capable of simultaneously implementing both interfaces, both
-currently as well as after adoption into Jakarta EE.
-
-A second interface, org.eclipse.microprofile.concurrent.ThreadContext,
-provides methods for individually contextualizing dependent stage
-actions. This gives the user more fine-grained control over the
-capture and propagation of thread context.
-It is intended that ThreadContext methods will one day be added to
-EE Concurrency's ContextService and for a single implementation to
-be capable of simultaneously implementing both interfaces, both
-currently as well as after adoption into Jakarta EE.
-
-[[builders]]
-Builders
-^^^^^^^^^
-
-It shall be possible to build instances of ManagedExecutor and
```

Comment:
"It shall be possible to ..." can change to, "With MicroProfile Context Propagation, you can ..."